src/scope/spout/sender.py

```diff
@@ -0,0 +1,308 @@
+"""
+Spout Sender - Sends textures to Spout receivers.
+
+This module provides a simple interface for sending textures to
+Spout-compatible applications like TouchDesigner, Resolume, etc.
+
+Note: The Python SpoutGL wrapper API differs from the C++ SDK.
+See: https://github.com/jlai/Python-SpoutGL
+"""
+
+import logging
+
+import numpy as np
+import torch
+
+logger = logging.getLogger(__name__)
+
+# Check if SpoutGL is available
+try:
+    import SpoutGL
+
+    SPOUT_AVAILABLE = True
+    # Log available methods for debugging
+    logger.debug(f"SpoutGL sender methods: {dir(SpoutGL.SpoutSender())}")
+except ImportError:
+    SPOUT_AVAILABLE = False
+    logger.warning("SpoutGL not available. Install with: pip install SpoutGL pyopengl")
```

Comment:
Let's add the `SpoutGL` and `pyopengl` deps (conditional on the platform being Windows) to `pyproject.toml` so they can be installed via `uv`.